ui/src/app/job-details/job-details.component.ts

```diff
@@ -38,6 +43,46 @@ export class JobDetailsComponent implements OnInit {
     });
   }
 
+  handleNavUp(): void {
+    if (this.job.extensions.parentJobId) {
+      this.router.navigate(['/jobs/' + this.job.extensions.parentJobId], {
+        queryParams: {
+          'q': this.route.snapshot.queryParams['q']
+        },
+        replaceUrl: true,
+        skipLocationChange: false
+      })
+      .then(result => {
+        this.handleNav();
```

Comment:
I was unable to figure out another way to navigate to a different instance of the job details view.  If you have an idea of a different implementation, maybe it would be best to handle in a new issue?

Reply:
That's fine if it's not part of this PR - I can't find it anymore. Was I looking at a bad diff?

Reply:
Not sure – it definitely shouldn't have shown up as part of this PR diff.